### PR TITLE
Feature/verification steps header

### PR DESCRIPTION
**Pull request by lufercc**
Summary
======
- Adding locator in Headder class
- add one verification to review user in the member button
- add Home and MemeberPopoerMenu class

### CODE REVIEW

File: src/main/java/com/webaut/project/pages/Header.java [hunk 1]
```diff
@@ -1,4 +1,32 @@
 package com.webaut.project.pages;
 
-public class Header {
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class Header extends AbstractPage{
```
Comment (lizzymendivil) [inline src/main/java/com/webaut/project/pages/Header.java]: Add a space between `AbstractPage` and the curly bracket please

File: src/main/java/com/webaut/project/pages/MemberPopoverMenu.java [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package com.webaut.project.pages;
+
+public class MemberPopoverMenu extends AbstractPage{
```
Comment (lizzymendivil) [inline src/main/java/com/webaut/project/pages/MemberPopoverMenu.java]: The same as in the previous comment

File: src/test/java/com/webaut/project/test/HeaderStep.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+import cucumber.api.java.en.Then;
+import org.testng.Assert;
+
+public class HeaderStep {
+
+    @Then("I verify the user {string} is in member button")
+    public void iVerifyTheUserIsInMemberButton(String userName) {
+        Header header= new Header();
+        Assert.assertTrue(header.userNameIsOnButton(userName), "user name founds");
+
```
Comment (lizzymendivil) [inline src/test/java/com/webaut/project/test/HeaderStep.java:13]: Remove this blank line

File: src/test/java/com/webaut/project/test/HeaderStep.java [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    }
+
```
Comment (lizzymendivil) [inline src/test/java/com/webaut/project/test/HeaderStep.java:15]: Remove this blank line

File: src/test/resources/features/login.feature [hunk 5]
```diff
@@ -1,4 +1,5 @@
 Feature: login
   Scenario: login and start a session
 
-    Given I login as "luiscachi" user with password "Control123"
\ No newline at end of file
+    Given I login as "luiscachi" user with password "Control123"
```
Comment (lizzymendivil) [inline src/test/resources/features/login.feature]: You shouldn't push your real credentials

**Comment by lizzymendivil** [comment]
Review done. Please check out my comments.

**Comment by lufercc** [comment]
Sorry Lizzy but I can't found your comments

**Comment by lufercc** [comment]
Hello, I resolved the observations, please check it,
also as notice, the TC doesn't execute success when the change the locators to using the # as Carlos mention

**Pull request merged**